www-servers/nginx-unit/nginx-unit-1.10.0.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python2_7 python3_{5,6,7})
+
+inherit python-single-r1
+
+MY_P="unit-${PV}"
+DESCRIPTION="A dynamic web and application server"
+HOMEPAGE="https://unit.nginx.org"
+SRC_URI="https://unit.nginx.org/download/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+MY_USE="perl python ruby"
+MY_USE_PHP="php5-6 php7-1 php7-2 php7-3"
+IUSE="${MY_USE} ${MY_USE_PHP}"
+REQUIRED_USE="|| ( ${IUSE} ) python? ( ${PYTHON_REQUIRED_USE} )"
```

Comment:
Could you tell me what this does exactly?

Reply:
This is meant to force the user to select at least one of the supported runtime/language support modules. It would be technically possible to build Unit without such a module, but there would not be any point to it, because no application could be run.

Reply:
Right, the `python? ( ${PYTHON_REQUIRED_USE} )` should probably be on its own line because it fooled me here for a moment?

Reply:
I split the definition into two lines.

Reply:
Thanks :)